components/prism-renpy.js

```diff
@@ -0,0 +1,25 @@
+Prism.languages.renpy= {
+
+	'string': /"""[\s\S]+?"""|'''[\s\S]+?'''|("|')(\\?.)*?\1|(^#?(?:(?:[0-9a-fA-F]{2}){3}|(?:[0-9a-fA-F]){3})$)/m,
+
+	'comment': {
+		pattern: /(^|[^\\])#.*?(\r?\n|$)/,
```

Comment:
Here the whole part `.*?(\r?\n|$)` can be replaced with `.+`, since I'm pretty sure we don't care about consuming the line feed.